pydatalab/src/pydatalab/config.py

```diff
@@ -239,6 +239,10 @@ class ServerConfig(BaseSettings):
 its importance when deploying a datalab instance.""",
     )
 
+    MAX_BATCH_CREATE_SIZE: int = Field(
+        100, description="Maximum number of items that can be created in a single batch operation."
```

Comment:
From the API side we can be much more lax, e.g., 10,000... just in case that is necessary. The main problem is that the UI is too slow for it atm, but it is a good idea to add an upper limit for the API anyway.

```suggestion
        10_000, description="Maximum number of items that can be created in a single batch operation."
```